.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "tools/OpenROAD"]
 	path = tools/OpenROAD
 	url = ../OpenROAD.git
+[submodule "tools/LSOracle-Plugin"]
+	path = tools/LSOracle-Plugin
+	url = https://github.com/lnis-uofu/LSOracle-Plugin.git
```

Comment:
I see this has yosys as a submodule which means we will have yosys twice.  Can we avoid that?

Reply:
It depends on how you are checking out the submodules. If the command is `git submodule init --recursive`, it will check it out in both places. The plugin just needs to be linked against the yosys header files, and the makefile for the plugin can be configured to point at any yosys include directory.